test/accessibility-rules.test.js

```diff
@@ -1,21 +1,36 @@
-const markdownlint = require("markdownlint");
-const accessibilityRulesConfig = require("../style/accessibility.json");
-const accessibilityRules = require("..");
+import { lint } from "markdownlint/async";
+import { githubMarkdownLint } from "../src/rules";
 
 const exampleFileName = "./test/example.md";
 const options = {
   config: {
+    "no-duplicate-heading": true,
+    "ol-prefix": "ordered",
+    "no-space-in-links": false,
+    "single-h1": true,
+    "no-emphasis-as-heading": true,
+    "no-empty-alt-text": false,
+    "heading-increment": true,
+    "no-generic-link-text": true,
+    "ul-style": {
+      style: "asterisk",
+    },
     default: false,
```

Comment:
Does `default: false` turn off all the rules by default-enabled rules?
I'm wondering why we need to explicitly set individual rules to `false` if `default: false`.